cypress/integration/basic-functionalities/local-tests/server/3_Change_your_password_functionality.spec.js

```diff
@@ -6,60 +6,69 @@ const newPassword = Cypress.env('user').newPassword
 
 // When running this test make sure that the base url is set to localhost:3000
 describe('Test for checking the change password functionality', () => {
-  context('Test for changing the password of the user', () => {
-    it('should be able to change the password of the user', () => {
-      cy.login({ email, password })
+  context('Test for changing the password of the user with misconfiguration or missing data', () => {
+    it('should not be able to change the password of the user with no password entered', () => {
+      cy.login({ email, password}) 
```

Comment:
cy.login({ email, password }); notice there's a blank space between password and }
Please, apply this everywhere.